src/jepsen/voltdb.clj

```diff
@@ -120,7 +119,6 @@
   "run voltdb init"
   [node]
   (info "Initializing voltdb")
-  (c/sudo username
           (c/cd base-dir
```

Comment:
whithout this sudo, will it work on Kyle's amazon machines?

Reply:
the premise is it's run's as the user who starts it, so you may need to adjust that depending on permissions.

Reply:
replaced with ";"